tests/integration-tests/tests/storage/kms_key_factory.py

```diff
@@ -0,0 +1,207 @@
+import json
+import logging
+import random
+import string
+import time
+
+import boto3
+import pkg_resources
+from jinja2 import Environment, FileSystemLoader
+
+
+class KMSKeyFactory:
+    """Manage creation for kms key."""
+
+    def __init__(self):
+        self.iam_client = None
+        self.kms_client = None
+        self.kms_key_id = None
+        self.account_id = boto3.client("sts").get_caller_identity().get("Account")
+        self.region = boto3.session.Session().region_name
```

Comment:
boto3 is getting this from the environment variable `AWS_DEFAULT_REGION`, right? Is there a reason to not use that?

Reply:
After the change, I am using it from the region fixture